config/metrics/crd-metrics-config.yaml

```diff
@@ -0,0 +1,676 @@
+---
+kind: CustomResourceStateMetrics
+spec:
+  resources:
+  - errorLogV: 0
+    groupVersionKind:
+      group: infrastructure.cluster.x-k8s.io
+      kind: VSphereCluster
+      version: v1beta1
+    labelsFromPath:
+      cluster_name:
+      - metadata
+      - labels
+      - cluster.x-k8s.io/cluster-name
+      name:
+      - metadata
+      - name
+      namespace:
+      - metadata
+      - namespace
+      uid:
+      - metadata
+      - uid
+    metricNamePrefix: capv_vspherecluster
+    metrics:
+    - each:
+        info:
+          labelFromKey: ""
```

Comment:
Why do we need this? Same below for nilIsZero, resourcePlural. Is there an omitempty missing for generation? Or is it intentional?

Reply:
Same for errorLogV? (what is this doing? :))

Reply:
Jep, it is missing omitempty.

errorLogV `	// ErrorLogV defines the verbosity threshold for errors logged for this resource.` which is the log level for klog.

Reply:
Thx!